src/ESLint.Formatter/sarif.js

```diff
@@ -54,7 +54,7 @@ module.exports = function (results, data) {
 
     const sarifLog = {
         version: "2.1.0",
-        $schema: "http://json.schemastore.org/sarif-2.1.0-rtm.5",
+        $schema: "http://json.schemastore.org/sarif-2.1.0-rtm.6",
```

Comment:
>$schema [](http://example.com/codeflow?start=8&length=7)

It's a very small thing but could this constant value be shared somewhere, so that we only make a single change, that flows to both the production formatter and its tests?